projects/RabbitMQ.Client/client/logging/RabbitMqClientEventSource.cs

```diff
@@ -34,25 +34,20 @@
 
 namespace RabbitMQ.Client.Logging
 {
-    [EventSource(Name="rabbitmq-dotnet-client")]
-    public sealed class RabbitMqClientEventSource : EventSource
+    #nullable enable
+    internal sealed partial class RabbitMqClientEventSource : EventSource
```

Comment:
I changed it to internal, as others shouldn't be able to do anything with these methods here